packages/logger/src/Logger.ts

```diff
@@ -352,15 +352,19 @@ export class Logger {
      * @returns {string} - file and line number
      */
     private getCallerFileAndLineTag(): string {
-        const frame: StackTrace.StackFrame[] = StackTrace.parse(new Error());
-        let callerStackIndex = 1;
-        while (frame[callerStackIndex].getFileName().indexOf(path.basename(__filename)) >= 0) {
-            // go up the stack until we're outside of the BrightsideLogger file
-            callerStackIndex += 1;
+        try {
+            const frame: StackTrace.StackFrame[] = StackTrace.parse(new Error());
+            let callerStackIndex = 1;
+            while (!frame[callerStackIndex].getFileName() || (frame[callerStackIndex].getFileName().indexOf(path.basename(__filename)) >= 0)) {
```

Comment:
@dkelosky Not sure if it's safe to assume that `frame[callerStackIndex].getFileName()` will always evaluate to a truthy value eventually? If there is a case where it doesn't, this seems like it could cause an infinite loop.